app/controllers/api/v1/books_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
 
       def show
         @book = Book.find(params[:id])
-        render json: @book
+        render_paginated json: @book
```

Comment:
`Book.find(id)` always return a single value, why did you use pagination on it ?

Reply:
done!